CONTRIBUTING.rst

```diff
@@ -99,7 +99,7 @@ Before you submit a pull request, check that it meets these guidelines:
 2. If the pull request adds functionality, the docs should be updated. Put
    your new functionality into a function with a docstring, and add the
    feature to the list in README.rst.
-3. The pull request should work for Python 2.7 and 3.4. Check
```

Comment:
Are we deprecating Python v2.x support for the `infoblox-client`?

Reply:
Yes, we are as future python versions are deprecating the 2.x support.